go.mod

```diff
@@ -24,7 +24,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-cty-funcs v0.0.0-20230405223818-a090f58aa992
 	github.com/hashicorp/hcl/v2 v2.19.1
-	github.com/moby/buildkit v0.13.0-beta1.0.20231129145300-c9ee8491d74f // master (v0.13.0-dev)
+	github.com/moby/buildkit v0.13.0-beta1.0.20231218103508-bbb6aaccb9ae // master (v0.13.0-dev)
```

Comment:
Can you vendor latest changes so it brings https://github.com/moby/buildkit/pull/4496?

Reply:
will do